domain/src/main/kotlin/no/nav/su/se/bakover/domain/grunnlag/SjekkOmGrunnlagErKonsistent.kt

```diff
@@ -0,0 +1,100 @@
+package no.nav.su.se.bakover.domain.grunnlag
+
+import arrow.core.Either
+import arrow.core.left
+import arrow.core.right
+import arrow.core.separateEither
+
+data class SjekkOmGrunnlagErKonsistent(
```

Comment:
https://ordbok.uib.no/perl/ordbok.cgi?ant_bokmaal=5&ant_nynorsk=5&ordbok=begge&OPP=konsistens

Reply:
Ikke så veldig nøye for meg egentlig, men konsistent treffer ganske bra for deler av det jeg ønsker å kommunisere, nemlig at det er motsigelsesfritt.
> i overført betydning: konsekvent, motsigelsesfri
konsistent argumentering

Kan vel diskuteres om "mangel på uføregrunnlag" er en motsigelse, så her blir kanskje konsistent litt for spesifikt. Konsekvens føler jeg treffer enda dårligere?